apps/api/src/services/data-provider/data-enhancer/yahoo-finance/yahoo-finance.service.ts

```diff
@@ -228,6 +228,11 @@ export class YahooFinanceDataEnhancerService implements DataEnhancerInterface {
             { name: assetProfile.summaryProfile?.sector, weight: 1 }
           ];
         }
+      } else if (assetSubClass == AssetSubClass.CRYPTOCURRENCY) {
```

Comment:
Can you remove this block and try to replace the following line above

```
response.symbol = assetProfile.price.symbol;
```

with

```
response.symbol = this.convertFromYahooFinanceSymbol(assetProfile.price.symbol);
```

Reply:
Solved